src/infrastructure/content/ContentFactory.java

```diff
@@ -1,19 +1,36 @@
 package infrastructure.content;
 
+/**
+ * This is a factory class of content module which contains instantiation of
+ * ContentCommunicator class and ServerPort class in two different methods from
+ * which other modules can get the object of those classes.
+ * @author Badal Kumar (111701008)
+ */
 public final class ContentFactory {
 	
```

Comment:
The server class needs to be added here too

Reply:
The instantiation of server class in content factory has been done in 94-ContentServer branch.

Reply:
If there are any merge conflicts, i'll raise an issue for you to fix it